### DISCUSSION TITLE
Unsupported code rate > 11/12

### DISCUSSION THREAD

**Discussion by pustovale** [NVlabs/sionna#147] [Q&A] [answered]
Dear developers,

I have a question what is the reason Sionna does not support LDPC code rate > 11/12. This prevents from running top mcs, e.g. 
mcs=27 from table 2.  There is assert in encoding.py

```
if self._coderate>(11/12):
             raise ValueError(
                     f"Unsupported coderate (r>11/12); n={n}, k={k}.")
```

After disabling this assert I was able to run mcs=27 from table 2. But it is not clear for me why the assert was set in the first place.

**Comment by SebastianCa** [comment, marked as answer]
Hi,

thanks for pointing this out. We will add support for `mcs=27` (i.e., `r<=948/1024`) in the next release of Sionna.
